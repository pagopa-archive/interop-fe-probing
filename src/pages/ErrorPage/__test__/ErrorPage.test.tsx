import { describe, expect, test } from 'vitest'
import { render } from '@testing-library/react'
import Errorpage from '../ErrorPage'

describe('ErrorPage', () => {
  test('render component', () => {
    const { container } = render(<Errorpage />)
    expect(container).toMatchSnapshot()
  })
})
