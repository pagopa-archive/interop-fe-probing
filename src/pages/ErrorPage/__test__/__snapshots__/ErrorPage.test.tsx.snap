// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ErrorPage > render component 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column css-1mm5gp4-MuiGrid-root"
  >
    <div
      class="MuiGrid-root MuiGrid-item css-bjrcl7-MuiGrid-root"
    >
      <h4
        class="MuiTypography-root MuiTypography-h4 css-5lbw0b-MuiTypography-root"
      >
         
        title
      </h4>
      <p
        class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
      >
        subtitle
      </p>
    </div>
  </div>
</div>
`;
